extractor/src/test/java/org/schabi/newpipe/extractor/services/peertube/PeertubeCommentsExtractorTest.java

```diff
@@ -14,10 +14,9 @@
 
 import java.io.IOException;
 import java.util.List;
+import java.util.Optional;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Do we really care about imports? What we should do is distribute IDE settings to prevent these kind of automatic refactorings. Resolving review comments about imports just wastes time IMO.